.github/workflows/ci.yml

```diff
@@ -54,8 +47,9 @@ jobs:
           bundler-cache: true
 
       - name: Configure JDK
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
+          distribution: 'zulu'
```

Comment:
We always want it to be `zulu`, even if we aren't sure what machine it's running on?

Reply:
This is actually the only distribution supported before, in the new version of the action, they added support for more and I just had to specify the one we used before.

Reply:
Oh fair enough, sounds good!